testing/e2e/stand_alone_test.go

```diff
@@ -155,94 +156,91 @@ func (suite *StandAloneSuite) TestClientAPI() {
 
 	enrollmentKey := suite.GetEnrollmentTokenForPolicyID(bCtx, "dummy-policy")
 
-	// Run subtests here
-	suite.Run("test status unauthenicated", func() {
-		ctx, cancel := context.WithCancel(bCtx)
-		defer cancel()
-		tester := &ClientAPITester{
-			suite.Suite,
-			ctx,
-			suite.client,
-			"https://localhost:8220",
+	versions := []string{"current", "2022-06-01"}
```

Comment:
Run the test for each the version here, if we add a new version we will have to create a new `api_version/client_api*.go` (probably worth adding this to the readme)